src/picosvg/svg_types.py

```diff
@@ -285,15 +286,20 @@ def callback(curr_xy, cmd, args, prev_xy, prev_cmd, prev_args)
                 if y_coord_idxs:
                     new_y += new_cmd_args[y_coord_idxs[-1]]
 
+                # z takes the pen to the return_pos
+                if new_cmd.lower() == "z":
+                    new_x, new_y = return_pos
+
                 prev_pos = copy.copy(curr_pos)
                 curr_pos = Point(new_x, new_y)
+                if new_cmd.upper() == "M":
+                    return_pos = copy.copy(curr_pos)
```

Comment:
I believe picosvg Points are immutable tuples so we don't need the extra copy.

Reply:
Fixed